src/worldofzuul/Room.java

```diff
@@ -11,22 +11,48 @@
  */
 public class Room {
 
+    /**
+     * The String "description" is used when describing the room generated with the constructor
+     * The HashMap is a list of existing rooms
+     */
     private String description;
     private HashMap<String, Room> exits;
 
+    /**
+     * A constructor for new rooms
+     * @param description 
+     * Description is the description of the new room
+     */
     public Room(String description) {
         this.description = description;
         exits = new HashMap<String, Room>();
     }
-
+    
+    /**
+     * The method sets the exits for the current room
+     * @param direction 
+     * Direction is the direction of the other rooms
```

Comment:
Skal flyttes en linje op så.
```
@param direction Direction is the direction of the other rooms
```

Reply:
Fixed in 20f03989dec3f91f64cdacc607da09352c372dcf